src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -80,21 +80,20 @@ export const createNoteAction = ctx => (onMouseDown, onMouseMove) => {
   let startPosition
   let startNote
 
-  onMouseDown(e => {
-    const pos = getLocal(e)
-    startPosition = pos
+  onMouseDown(local => {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->